_includes/selector.html

```diff
@@ -705,22 +733,26 @@
             releaseClickHandler(e, release) {
                 if (this.isDisabled(e.target)) return;
                 this.active_release = release;
-                if ( this.active_release === "Stable" && this.active_cuda_ver === "12.5") {
-                    this.active_cuda_ver = "12.2";
+                if (this.active_release === "Stable" && this.active_docker_cuda_ver === "12.5") {
+                    this.active_docker_cuda_ver = "12.2";
                 }
```

Comment:
What do you think about capturing `12.5` & `12.2` in variables in the "all possible values" section?

Perhaps we could do something like...?

```suggestion
                if (this.active_release === "Stable" && this.active_docker_cuda_ver === this.nightly_docker_cuda_ver) {
                    this.active_docker_cuda_ver = this.stable_docker_cuda_ver;
                }
```

Reply:
Could we leave this for a later PR? I think this ties in with the [above thread](https://github.com/rapidsai/docs/pull/524#discussion_r1688249043), where I struggled to get a "structured" JSON format to work.